enforcer.go

```diff
@@ -416,6 +425,24 @@ func (e *Enforcer) enforce(matcher string, rvals ...interface{}) (ok bool, err e
 
 			parameters.pVals = pvals
 
+			if hasEval {
+				ruleName, err := util.GetEvalValue(expString)
```

Comment:
how do you handle multiple evals?

Reply:
sorry, I forgot about this situation, now it can be handled correctly